src/mapgen.cpp

```diff
@@ -292,9 +293,10 @@ void Mapgen::calcLighting(v3s16 nmin, v3s16 nmax)
 
 	//printf("updateLighting: %dms\n", t.stop());
 }
+*/
```

Comment:
If you've decided to remove this version of calcLighting, why keep it commented out?